test/apis/batch/inferentia/cortex_inf.yaml

```diff
@@ -0,0 +1,14 @@
+- name: image-classifier-resnet50
+  kind: BatchAPI
+  handler:
+    type: python
+    path: handler.py
+    config:
+      model_path: s3://cortex-examples/pytorch/image-classifier-resnet50
```

Comment:
Add a README/comment explaining that the highest batch size can only be 1 due to how the Inferentia model was compiled.

Reply:
this is under the `test` directory, so I don't think it is important to add a README. I can add the comment somewhere 👍